frontend/public/components/_overview.scss

```diff
@@ -26,8 +26,72 @@
 }
 
 .overview {
-  width: 100%;
+  bottom: 0;
   display: flex;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 67px;
+  @media(min-width: $screen-sm-min) {
+    left: 220px;
+    top: 100px;
```

Comment:
Should use a var.

Global notifications are currently not in use (impersonate user is currently disabled), but this will be problematic if they are re-enabled.